### PR TITLE
Fixes no root centos 7

### PR DESCRIPTION
**Pull request by sergiopena**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,15 @@ with the [hostnames](http://community.opscode.com/cookbooks/hostnames) cookbook
...
 
-In order to test the cookbook you will need to install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/).
+In order to test the cookbook you will need to install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/). 
+
+* Tested on:
+
+| Operating System | Vagrant version | VirtualBox version |
+|---|---|---|
+| Fedora 25 |  1.8.5 | 5.1.14r112924 |
+| OS X 10.12.2 | 1.9.1 | 5.0.32r112930 |
```
Comment (nacx) [inline README.md]: OSX: 10.12.3
VirtualBox: 5.0.42r108355
Vagrant: 1.8.4

Comment (chirauki) [inline README.md]: Should we actually keep such a detailed list?

**Comment by nacx** [comment]
I'd also add an integration test to the `config` suites that check the changed files, and add something like:

```ruby
expect(file('/opt/abiquo/tomcat/conf/server.xml')).to be_owned_by('tomcat')
```

**Comment by nacx** [comment]
Also, does the `abiquo.properties` file need to be owned by `tomcat` too?

File: README.md (hunk 1 above)
Comment (nacx) [inline README.md]: We don't need a detailed list definitely, but there are some combinations of vagrant and vbox that are incompatible and you get meaningless errors when running the kitchen tests. We should provide at lease one or some officially supported versions.

**Comment by sergiopena** [comment]
Tomcat does not claim about ownership of abiquo.properties but owned of tomcat pid must own the server.xml

**Pull request merged**